keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -65,7 +65,7 @@
             }
           },
           "default": {
-            "description": "Default response. It will be deserialized as per the error definition specified in the schema. An exception will be thrown.",
+            "description": "Default response. It will be deserialized per the error definition specified in the schema. An exception will be thrown.",
```

Comment:
@johanste and @pomortaz - thanks for your thoughts Johan. Documenting via swagger is new to me so, I'm eager to understand alternative. I agree this needs a better explanation that suites the cases of both REST and client API. Pooneh, what suggestions do you have on this?

I'm going to add in my local expect on swagger docs, @BryanLa, looking for suggestions. Bryan, how have you seen this handled in other swagger docs cases?